atd-etl/cr3_download/cr3_download.py

```diff
@@ -55,21 +48,28 @@ def wait(int):
     response = get_crash_id_list(downloads_per_run=downloads_per_run)
     print("\nResponse from Hasura: %s" % json.dumps(response))
 
-    crashes_list = response['data']['atd_txdot_crashes']
+    crashes_list = response["data"]["atd_txdot_crashes"]
 
     crashes_list_without_skips = [
         x for x in crashes_list if x["crash_id"] not in known_skips
     ]
-    print("\nList of crashes needing CR3 download: %s" % json.dumps(crashes_list_without_skips))
- 
+    print(
+        "\nList of crashes needing CR3 download: %s"
+        % json.dumps(crashes_list_without_skips)
+    )
+
```

Comment:
might just be my personal preference, but i think it'd be nice if this instead (or additionally) logs the **number** of crashes needing download.

Reply:
I agree that the count is useful. I'm adding that 🙏